odd-platform-ui/src/components/DataQuality/DataQualityFilters/DataQualityFiltersStyles.ts

```diff
@@ -0,0 +1,15 @@
+import { Grid } from '@mui/material';
+import styled from 'styled-components';
+
+export const Container = styled('div')(({ theme }) => ({
+  padding: theme.spacing(2, 1, 1.5, 1),
+}));
+
+export const FacetsLoaderContainer = styled(Grid)(({ theme }) => ({
+  padding: theme.spacing(0, 3),
+  justifyContent: 'center',
+}));
+
+export const ListContainer = styled('div')(() => ({
+  margin: '16px 0',
```

Comment:
let's stick to `css` function. And we have spacing step pattern with 8px somewhere in palette.ts For example:
```
const ListContainer = styled('section')(
  ({ theme }) => css`
    margin: ${theme.spacing(2)} ${theme.spacing(0)};
  `
);
```